test/e2e_node/cpu_manager_test.go

```diff
@@ -183,15 +183,17 @@ func runCPUManagerTests(f *framework.Framework) {
 	var pod, pod1, pod2 *v1.Pod
 
 	It("should assign CPUs as expected based on the Pod spec", func() {
-		oldCfg = enableCPUManagerInKubelet(f)
-
 		cpuCap, cpuAlloc, cpuRes = getLocalNodeCPUDetails(f)
 
-		// Skip CPU Manager tests if the number of allocatable CPUs < 1.
-		if cpuAlloc < 1 {
-			framework.Skipf("Skipping CPU Manager tests since the number of allocatable CPUs < 1")
+		// Skip CPU Manager tests altogether if the number of allocatable
+		// CPUs < 2.
```

Comment:
Could we add to this comment: this assumes no kube-reserved or system-reserved is configured before we call `enableCPUManagerInKubelet(...)`

Reply:
Actually on second look, why not just use capacity? Basing the decision on allocatable doesn’t make a ton of sense because we immediately restart Kubelet with a new kube-reserved amount. In other words if

getLocalNode(f).Status.Capacity[v1.ResourceCPU]

is < 2, then skip.

Reply:
The other condition on L#293 should be updated to match.

Reply:
Ok. I will change it to look at capacity. But FWIW, at this point in the tests, capacity will almost always be equal to allocatable. 
In L#293, the condition should not change as it needs to look at allocatable and allocatable needs to be >= 2.